Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -2346,6 +2346,19 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     return configuration;
 }
 
+WebsiteDataStore* WebExtensionContext::websiteDataStore(std::optional<PAL::SessionID> sessionID) const
+{
+    RefPtr result = extensionController()->websiteDataStore(sessionID);
+    if (result && !result->isPersistent() && !hasAccessInPrivateBrowsing())
+        return nullptr;
+    return result.get();
+}
+
+void WebExtensionContext::cookiesDidChange(API::HTTPCookieStore& cookieStore)
```

Comment:
should we be doing something with `cookieStore` here?

Reply:
We can't. I'll put a FIXME here too.